parse/arrival-or-departure.js

```diff
@@ -27,11 +27,19 @@ const createParseArrOrDep = (prefix) => {
 			stop: d.stbStop.location || null,
 			...profile.parseWhen(ctx, d.date, tPlanned, tPrognosed, tzOffset, cancelled),
 			...profile.parsePlatform(ctx, plPlanned, plPrognosed, cancelled),
-			// todo: for arrivals, this is the *origin*, not the *direction*
-			direction: prefix === DEPARTURE && d.dirTxt && profile.parseStationName(ctx, d.dirTxt) || null,
-			provenance: prefix === ARRIVAL && d.dirTxt && profile.parseStationName(ctx, d.dirTxt) || null,
+			headsign: d.dirTxt && profile.parseStationName(ctx, d.dirTxt) || null,
```

Comment:
Removing a field, or changing its type signature (e.g. from "string or `null`" to "stop object or `null`") is considered a breaking change in `hafas-client`.